python/GafferSceneUI/TransformQueryUI.py

```diff
@@ -0,0 +1,129 @@
+import Gaffer
+import GafferScene
+
+Gaffer.Metadata.registerNode(
+
+    GafferScene.TransformQuery,
+
+    "description",
+    """
+    Queries a particular location in a scene and outputs the transform.
+    """,
+
+    # Work around StandardNodeGadget layout bug that would
+	# place the `filter` nodule inside the frame.
+	"nodeGadget:minWidth", 12.0,
+
+    plugs = {
+
+        "scene" : [
+
+            "description",
+            """
+            The scene to query the transform for.
+            """,
+
+        ],
+
+        "location" : [
+
+			"description",
+			"""
+			The location within the scene to query the transform at.
+
+			> Note : If the location does not exist then the query will not be
+			> performed and all outputs will be set to their default values.
+			""",
+
+			"plugValueWidget:type", "GafferSceneUI.ScenePathPlugValueWidget",
+			"scenePathPlugValueWidget:scene", "scene",
+			"nodule:type", "",
+
+		],
+
+        "space" : [
+
+            "description",
+            """
+            The space to query the transform.
+            """,
+
+            "preset:Local", GafferScene.TransformQuery.Space.Local,
+            "preset:World", GafferScene.TransformQuery.Space.World,
+            "preset:Relative", GafferScene.TransformQuery.Space.Relative,
+
+            "plugValueWidget:type", "GafferUI.PresetsPlugValueWidget",
+        ],
```

Comment:
I think it's probably worth having `"nodule:type", ""` here, and for the `invert` plug, so that they're not displayed in the GraphEditor.